ui/app/AppLayouts/Wallet/services/dapps/DAppsListProvider.qml

```diff
@@ -37,6 +58,19 @@ QObject {
 
                 let dappsList = JSON.parse(dappsJson);
                 for (let i = 0; i < dappsList.length; i++) {
+                    const cachedEntry = dappsList[i];
```

Comment:
Could you please add a disconnect as the first call in this function?
`root.store.dappsListReceived.disconnect(dappsListReceivedFn);`

Reply:
Indeed, from higher perspective it makes sense as `getActiveSessionsFn` is a one-time slot to get a local cache of stored dApps.